### DISCUSSION TITLE
Show only values after a date, eg >= now()

### DISCUSSION THREAD

**Discussion by deltamelter** [kalkih/mini-graph-card#980] [Q&A] [answered]
I see similar discussions but not understanding a solution that would let me take data from a half-hourly sensor with last 24 hours and next 8-12 forecasted and only graph future values including the current hour and maybe previous 1-2 hours for context.
Currently the graph only shows data from "the beginning" onwards. or last 24 hours up to now.   I want to see the future ;)

currently using/trying
```
hours_to_show: 24
points_per_hour: 2
cache: false
max_age:
  hours: 2

``` 

![Screenshot_20230623_093036_Home Assistant](https://github.com/kalkih/mini-graph-card/assets/27061628/17e09048-5557-4e76-a815-068d5c0afed6)

**Comment by ildar170975** [comment]
Mainly sensors providing forecasts uses some kind of arrays like this:
![image](https://github.com/kalkih/mini-graph-card/assets/71872483/5498ece0-3f6e-42c9-9c4f-c775db7a1280)
Here each forecast entry has a timestamp.
The card's X-axis is filled by timestamps of an entity changes (cannot be filled by values of some attributes).

**Reply by ildar170975** [reply]
Also, the card does not have these options from your code:
```
 max_age:
   hours: 2
```

**Reply by deltamelter** [reply]
The sensor contains an array of Rates from 24 hours ago to x hour from now (tonight).  I want this graph to show the current and future values only.  Is there a filter?  Sorry, fairly new to this.
max_age is just something I read about and tried just in case :)

**Reply by deltamelter** [reply]
the table in my screenshot is from the same sensor but using js code to only show current and future.  Trying to do this with graph.

**Reply by ildar170975** [reply]
Consider using ANOTHER graph which may represent these data nicer)).
Even custom:bar graph may be used:
![image](https://github.com/kalkih/mini-graph-card/assets/71872483/ba6d9dc7-bee5-44f0-a6ac-060464b116a6)

**Reply by deltamelter** [reply]
do "OTHER" graphs have the ability to choose the data range?  Not sure if you are telling me to "go away" and try a different graph solution?

**Reply by ildar170975** [reply, marked as answer]
Cannot answer for all "other cards". Just told you that

> The card's X-axis is filled by timestamps of an entity changes (cannot be filled by values of some attributes).

**Reply by deltamelter** [reply]
ok, wasn't looking for solution with other cards, just this mini-graph-card.  But if it can't be done, so be it.  Thanks for responding.

**Reply by deltamelter** [reply]
Thanks for directing me to look elsewhere...  
I found a more flexible graph card and achieved what I wanted.

**Reply by ildar170975** [reply]
Now you can share with people what you found, can't you?

**Reply by deltamelter** [reply]
![image](https://github.com/kalkih/mini-graph-card/assets/27061628/b0b2526f-e3fa-4a0e-a1ed-813d4683fe49)
https://github.com/RomRider/apexcharts-card
